addons/hw_posbox_homepage/static/src/app/components/dialog/HandlerDialog.js

```diff
@@ -0,0 +1,191 @@
+                                t-att-value="interface[1].is_using_parent_level ? 'parent' : interface[1].level">
+                                <option value="parent">Same as Odoo</option>
+                                <option value="info">Info</option>
+                                <option value="debug">Debug</option>
+                                <option value="warning">Warning</option>
+                                <option value="error">Error</option>
+                            </select>
+                            <input t-else="" type="text" class="form-control" aria-label="Text input with dropdown button" disabled="true" placeholder="Logger uninitialised" />
+                        </div>
+                    </div>
+                    <div class="mb-3">
+                        <h5>Drivers logs level</h5>
+                        <div t-foreach="Object.entries(this.state.handlerData.drivers_logger_info)" t-as="drivers" t-key="drivers[0]" class="input-group input-group-sm mb-3">
+                            <label class="input-group-text w-50" t-att-for="drivers[0]" t-esc="drivers[0]" />
+                            <select t-att-name="'iot-logging-driver-'+drivers[0]"
+                                t-if="drivers[1]"
+                                class="form-select"
+                                t-on-change="(ev) => this.onChange(ev.target.name, ev.target.value)"
+                                t-att-id="drivers[0]"
+                                t-att-value="drivers[1].is_using_parent_level ? 'parent' : drivers[1].level">
+                                <option value="parent">Same as Odoo</option>
+                                <option value="info">Info</option>
+                                <option value="debug">Debug</option>
+                                <option value="warning">Warning</option>
+                                <option value="error">Error</option>
+                            </select>
+                            <input t-else="" type="text" class="form-control" aria-label="Text input with dropdown button" disabled="true" placeholder="Logger uninitialised" />
+                        </div>
+                    </div>
+                    <div>
+                        <h5>Debug</h5>
+                        <div class="d-flex gap-2">
+                            <button class="btn btn-primary btn-sm" t-on-click="loadIotHandlers">
+                                Load IOT Handlers
+                            </button>
+                            <button class="btn btn-primary btn-sm" t-on-click="clearIotHandlers">
+                                Clear IOT Handlers
+                            </button>
+                        </div>
+                    </div>
+                </t>
+            </t>
+            <t t-set-slot="footer">
+                <button type="button" class="btn btn-primary btn-sm" data-bs-dismiss="modal">Close</button>
```

Comment:
when closing this specific dialog, a timer is briefly shown
![image](https://github.com/user-attachments/assets/c96928b9-771c-4722-b89f-53fb3ae8e5f5)